### PR TITLE
Add multichain cold key management

### LINKED ISSUE #71: MultiChain cold storage how-to
Write a MultiChain cold storage how-to and put it under `Operations` if #34 goes through

### PR DESCRIPTION
**Pull request by akegaviar**
Added instructions on signing tx on the mutlichain cold node
without exposing the private keys to the hot nodes.

Closes #71

### CODE REVIEW

File: docs/operations/multichain/cold-node-key-management.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+By default, you can view your MultiChain wallet addresses and keys specific to the node that you deployed, provided that you are logged in as the same user with Chainstack. See [View node access and credentials](/platform/view-node-access-and-credentials).
+
+You can use the default wallet addresses and key pairs for development and testing purposes.
+
+If you are moving your MultiChain network to production, you are strongly recommended to use secure key management.
+
+Cold node key management assumes that you do not store the keys in a hot MultiChain wallet. Instead, you have the keys stowed away in a Multichain wallet on a cold node.
+
+This section will guide you through setting up a Multichain cold node, creating a key pair, issuing an asset, and signing the transaction on the cold node without exposing the private key to the Multichain network.
```
Comment (easeev) [inline docs/operations/multichain/cold-node-key-management.md]: MultiChain and not Multichain here and everywhere

File: docs/operations/multichain/cold-node-key-management.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+2. Set up a Multichain cold node on the server
+
+   1. Create a `~/.multichain-cold` directory on the server
+
+   1. In the `~/.multichain-cold` directory, create a subdirectory with the chain name. See [View node access and credentials](/platform/view-node-access-and-credentials) to get the chain name of your Multichain network running with Chainstack.
+
+      Example: `~/.multichain-cold/nw-123-456-7`.
+
+   1. In the directory, create an empty `params.dat file`
```
Comment (easeev) [inline docs/operations/multichain/cold-node-key-management.md]: `params.dat` file?

File: docs/operations/multichain/cold-node-key-management.md [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+
+Example:
+
+``` sh
+curl https://nd-123-456-789.p2pify.com -u "user-name:pass-word-pass-word-pass-word" -d '{"method":"decoderawtransaction","params":["010000000177acc507ee0b802f3b0dde19f3acb389a40498f388822fb3d48efa871135f1f60000000000ffffffff0200000000000000003776a914aab5af995a8d50eb35ca7044880b4ccc463b7e8c88ac1c73706b71a40498f388822fb3d48efa871135f1f650c30000000000007500000000000000003776a91421722e2f3414baa7c29cc31e833cfb01bef2eece88ac1c73706b71a40498f388822fb3d48efa871135f1f650c30000000000007500000000"],"id":1}'
+```
+
+Example output:
+
+``` sh
```
Comment (easeev) [inline docs/operations/multichain/cold-node-key-management.md]: json?

File: docs/operations/multichain/cold-node-key-management.md [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+
+Example:
+
+``` sh
+curl https://nd-123-456-789.p2pify.com -u "user-name:pass-word-pass-word-pass-word" -d '{"method":"gettxout","params":["f6f1351187fa8ed4b32f8288f39804a489b3acf319de0d3b2f800bee07c5ac77",0],"id":1}'
+```
+
+Example output:
+
+``` sh
```
Comment (easeev) [inline docs/operations/multichain/cold-node-key-management.md]: json?

File: docs/operations/multichain/cold-node-key-management.md [hunk 5]
```diff
@@ -0,0 +1,395 @@
...
+
+       For the examples in this guide, the full command is:
+
+       ``` js
+       signrawtransaction 010000000177acc507ee0b802f3b0dde19f3acb389a40498f388822fb3d48efa871135f1f60000000000ffffffff0200000000000000003776a914aab5af995a8d50eb35ca7044880b4ccc463b7e8c88ac1c73706b71a40498f388822fb3d48efa871135f1f650c30000000000007500000000000000003776a91421722e2f3414baa7c29cc31e833cfb01bef2eece88ac1c73706b71a40498f388822fb3d48efa871135f1f650c30000000000007500000000 '[{"txid":"f6f1351187fa8ed4b32f8288f39804a489b3acf319de0d3b2f800bee07c5ac77","vout":0,"scriptPubKey":"76a91421722e2f3414baa7c29cc31e833cfb01bef2eece88ac0c73706b67a08601000000000075"}]'
+       ```
+
+       Example output:
+
+       ``` js
```
Comment (easeev) [inline docs/operations/multichain/cold-node-key-management.md]: json?

File: docs/operations/multichain/cold-node-key-management.md [hunk 6]
```diff
@@ -0,0 +1,395 @@
...
+      ```
+
+   1. Convert the output to the `params.dat` format
```
Comment (easeev) [inline docs/operations/multichain/cold-node-key-management.md]: Perhaps we can simplify this a little if we call `getblockchainparams` with, `display-names=false`? Ref. https://www.multichain.com/developers/json-rpc-api/

File: docs/operations/multichain/cold-node-key-management.md (hunk 1 above)
Comment (akegaviar) [reply docs/operations/multichain/cold-node-key-management.md]: This is embarassing. Fixed.

File: docs/operations/multichain/cold-node-key-management.md (hunk 2 above)
Comment (akegaviar) [reply docs/operations/multichain/cold-node-key-management.md]: My bad. Fixed.

File: docs/operations/multichain/cold-node-key-management.md (hunk 6 above)
Comment (akegaviar) [reply docs/operations/multichain/cold-node-key-management.md]: No. Just checked, and it prints the same thing but without the hyphens which is even worse.

Output example with `display-names=false`:

``` json
# curl https://nd-086-977-389.p2pify.com -u "upbeat-visvesvaraya:oven-drudge-recoup-crafty-setup-scoop" -d '{"method":"getblock
chainparams","params":[false],"id":3,"chain_name":"nw-614-792-6"}'
{"result":{"chainprotocol":"multichain","chaindescription":"My Network","rootstreamname":"root","rootstreamopen":true,"chainistestnet":false,"targetblocktime":15,"maximumblocksize":8388608,"maximumchunksize":1048576,"maximumchunkcount":1024,"defaultnetworkport":7447,"defaultrpcport":8000,"anyonecanconnect":false,"anyonecansend":false,"anyonecanreceive":false,"anyonecanreceiveempty":true,"anyonecancreate":false,"anyonecanissue":false,"anyonecanmine":false,"anyonecanactivate":false,"anyonecanadmin":false,"supportminerprecheck":true,"allowarbitraryoutputs":false,"allowp2shoutputs":true,"allowmultisigoutputs":true,"setupfirstblocks":60,"miningdiversity":0.3,"adminconsensusupgrade":0.5,"adminconsensustxfilter":0.5,"adminconsensusadmin":0.5,"adminconsensusactivate":0.5,"adminconsensusmine":0.5,"adminconsensuscreate":0,"adminconsensusissue":0,"lockadminminerounds":10,"miningrequirespeers":true,"mineemptyrounds":10,"miningturnover":0.5,"firstblockreward":-1,"initialblockreward":0,"rewardhalvinginterval":52560000,"rewardspendabledelay":1,"minimumperoutput":0,"maximumperoutput":100000000000000,"minimumoffchainfee":0,"minimumrelayfee":0,"nativecurrencymultiple":100000000,"skippowcheck":false,"powminimumbits":8,"targetadjustfreq":-1,"allowmindifficultyblocks":false,"onlyacceptstdtxs":true,"maxstdtxsize":4194304,"maxstdopreturnscount":32,"maxstdopreturnsize":2097152,"maxstdopdropscount":5,"maxstdelementsize":40000,"chainname":"nw-614-792-6","protocolversion":20004,"networkmessagestart":"f8d8cdf4","addresspubkeyhashversion":"007ee2f6","addressscripthashversion":"05d9757c","privatekeyversion":"80c92c0c","addresschecksumvalue":"08f94387","genesispubkey":"02fbe033371977dc3f22bc80aedcc10d72e273ed6443697b83375879506560322b","genesisversion":1,"genesistimestamp":1566290098,"genesisnbits":536936447,"genesisnonce":193,"genesispubkeyhash":"4a4aa22b2bab3a089943f1d4270bd5a4a04098ef","genesishash":"000d4dec342d6c1f871366232981c8e27efa08e6911f77728f324c29f81823e4","chainparamshash":"6ccfa0843096d9f60e23111d98418d24435504d4d4883e0d7224967c3150adf0"},"error":null,"id":3}
```

**Pull request merged**